packages/appsync-emulator-serverless/README.md

```diff
@@ -45,6 +45,14 @@ If using the DynamoDB emulator, data is preserved between emulator runs and is s
 # NOTE unless you assign a specific port a random one will be chosen.
 yarn appsync-emulator --port 62222
 ```
+
+#### subscriptions with fixed websocket port
+
+use `--ws-port` to set up a port for websocket(used for subscriptions) otherwise it will be any random free
```

Comment:
How do you feel about:

```
use `--ws-port` option to assign a websocket port used for graphql subscriptions, otherwise a random one will be chosen
```

?